httpmw/Ip_cognito_auth_test.go

```diff
@@ -45,17 +62,18 @@ func TestCognitoIPSucceed(t *testing.T) {
 func TestCognitoIP401(t *testing.T) {
 	called := false
 	assert := assert.New(t)
-	ipRanges := "192.168.10.1-192.168.10.10,192.168.20.1-192.168.20.10"
+	ipRanges := authTestIPRangesLarge
```

Comment:
This variable is redundant you can just pass `authTestIPRangesLarge` directly into  `IpCognitoAuth`.